app/controllers/renews_controller.rb

```diff
@@ -6,6 +6,10 @@ class RenewsController < ApplicationController
   def new
     authorize
 
+    # Back office only: Badly formed transient registrations sometimes cause renewals to fail.
+    # So we clear any existing transient registrations before starting a back-office renewal.
+    WasteExemptionsEngine::TransientRegistration.where(reference: registration.reference).destroy_all
```

Comment:
Is this approach safe? I've imagined a situation where I have two windows open. I've almost completed the renewal in one window, and then accidentally clicked renew in another. Will the renewal that is in progress get removed as a result?

Reply:
Yes I think it's sound.